cmd/kwild/config/default_config.toml

```diff
@@ -251,7 +251,7 @@ max_txs_bytes = 536870912
 max_tx_bytes = 4194304
 
 # Size of the cache (used to filter transactions we saw earlier) in transactions
-cache_size = 10000
+cache_size = 20000
```

Comment:
I think we should have higher cache_size here, atleast to the size of mempool. This is what keeps track of previously seen transactions and avoids rechecks and duplicate txs in the mempool.

Reply:
I would hope that a cache **miss** would just lead to a tx index check.  I'll increase this number, but do you have a link for how their mempool cache is works if there's a miss?  It's a badly cache-supported system that would result in duplicate txns in mempool in the event of a cache, so the purpose of this value doesn't seem clear.

Reply:
Bumped to 60,000 in https://github.com/kwilteam/kwil-db/pull/841/commits/a9ec473094c75cb8629e3e6f1a1830f51ce7aec9.
Will look into the functionality of their internal cache, but I think it's safer higher as you suggested.

Reply:
Yeah, else we might get into the hash recalculations, which we really wanna avoid as much as possible.